MapboxCoreNavigation/RouteStepFormatter.swift

```diff
@@ -23,15 +23,12 @@ public class RouteStepFormatter: Formatter {
         
         let modifyValueByKey = { (key: OSRMTextInstructions.TokenType, value: String) -> String in
             switch key {
-            case .code:
-                let refComponents = value.addingXMLEscapes.components(separatedBy: .whitespaces)
-                guard var firstRefComponent = refComponents.first else { return value.asSSMLAddress }
+            case .wayName, .destination, .rotaryName, .code:
+                let stringComponents = value.addingXMLEscapes.components(separatedBy: .whitespaces)
```

Comment:
FYI, this will miss any codes like `A-35`. OSM tagging conventions call for a space, as in `I 35`, but I do see some other regions using hyphens. We could harden this code by enumerating the string with `String.enumerateSubstrings(in:options:_:)` and `byWord`, or with `String.replacingOccurrences(of:with:options:range:)` and `regularExpression`.